aspnetcore/performance/response-compression/sample/Startup.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO.Compression;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.ResponseCompression;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
+
+namespace ResponseCompressionSample
+{
+    public class Startup
+    {
+        #region snippet2
+        public void ConfigureServices(IServiceCollection services)

```

Comment:
This snippet is used many times in the doc. Some of it may be unavoidable but let's try to split it up a little so it's more relevant to the different sections. Maybe services.AddResponseCompression and services.Configure<GzipCompressionProviderOptions> can probably be separate snippets?